learn/advanced/updating.md

```diff
@@ -20,6 +20,10 @@ This guide only works for versions v0.15 and above. If you are using an older ve
 
 This section contains instructions for upgrading from specific versions. Most versions don't require version-specific steps and you should be able to upgrade directly. If the version you are upgrading from isn't listed here, no additional steps are required.
 
+- If you are updating **from v0.19 or below**, please ensure all updates finish processing before creating the dump. `enqueued` updates will not be exported and may result in data loss.
```

Comment:
@dureuill I know you already reviewed this, but I wanted to double-check because it's a bit surprising!

Is this behavior specific to exporting a dump from Meilisearch versions < v0.20? Not including v0.20?

I'm surprised because I know that v0.21 is when we introduced v2 of dumps, so I would think that it would also affect v0.20.

Thanks 🙏🏻

Reply:
Thank you @dichotommy for the double-check!

You would be correct [I wrote versions <0.20](https://github.com/meilisearch/meilisearch/issues/2985#issuecomment-1333840498), but what I meant was really <=0.20. Every version of Meilisearch making use of the dumps v1 has the same issue of not exporting the necessary data to complete any pending task.

That must be why they say that the two hard problems of computing are naming, cache invalidation, and off-by-one errors 😬

Reply:
> That must be why they say that the two hard problems of computing are naming, cache invalidation, and off-by-one errors 😬

Hahaha, that's a good one 😆